src/status_im/ui2/screens/chat/composer/mentions.cljs

```diff
@@ -11,32 +11,33 @@
 (defn mention-item
   [[public-key {:keys [alias name nickname] :as user}] _ _ text-input-ref]
   (let [ens-name? (not= alias name)]
-    [rn/touchable-opacity {:on-press #(rf/dispatch [:chat.ui/select-mention text-input-ref user])}
+    [rn/touchable-opacity
+     {:on-press            #(rf/dispatch [:chat.ui/select-mention text-input-ref user])
+      :accessibility-label (str "Press to mention " (or nickname alias nickname) " in your message")}
```

Comment:
we use `accessibility-label` only for e2e, so it should be a keyword, like `:mention-item` cc @churik